raaghu-elements/src/rds-toast/rds-toast.stories.tsx

```diff
@@ -59,54 +60,124 @@ type Story = StoryObj<typeof RdsToast>;
+        autohide: false,    
         colorVariant: "light",
+        showHeader: true,
+        withIcon: true,
+        iconName: "circle",
         iconColorvariant: "primary",
         iconHeight: "18px",
         iconWidth: "18px",
-        iconFill: false
+        iconFill: false,
+        borderColor: "primary",
+        layout : "download",
+        state: "basic",
+        progressWidth: 40,
+        filename: "Filename.txt"
     }
 } satisfies Story;
-toastWithAutohide.parameters = { controls: { include: ['headerTitle', 'message', 'delay', 'autohide', 'withIcon', 'showHeader', 'iconName', 'colorVariant', 'iconColorvariant', 'iconHeight', 'iconWidth', 'iconFill'] } };
+toastWithDownload.parameters = { controls: { include: ["headerTitle", "message", "delay", "autohide", "withIcon", "showHeader", "iconName", "iconHeight", "iconWidth", "iconFill", "borderColor", "layout", "state", "progressWidth", "filename"] } };
 
-export const toastWithoutHeader: Story = {
+export const toastWithChat: Story = {
     args: {
-        headerTitle: "Toast",
-        autohide: false,
+        headerTitle: "Toast Headline",
+        message: "This is a help text",
+        delay: 5000,
+        autohide: false,    
+        colorVariant: "light",
+        showHeader: true,
         withIcon: true,
+        iconName: "circle",
+        iconColorvariant: "primary",
+        iconHeight: "18px",
+        iconWidth: "18px",
+        iconFill: false,
+        borderColor: "primary",
+        layout : "chat",
+        state: "basic",
+        placeholder: "Placeholder Text"
+    }
+} satisfies Story;
+toastWithChat.parameters = { controls: { include: ["headerTitle", "message", "delay", "autohide", "withIcon", "showHeader", "iconName", "iconHeight", "iconWidth", "iconFill", "borderColor", "layout", "placeholder"] } };
+
+export const toastWithRequest: Story = {
+    args: {
+        headerTitle: "Toast Headline",
+        message: "This is a big sample placeholder text.",
         delay: 5000,
-        showHeader: false,
-        message: "This is a sample toast",
+        autohide: false,    
         colorVariant: "light",
-        iconName: "folder",
+        showHeader: true,
+        withIcon: true,
+        iconName: "circle",
         iconColorvariant: "primary",
         iconHeight: "18px",
         iconWidth: "18px",
-        iconFill: false
+        iconFill: false,
+        borderColor: "primary",
+        layout : "request",
+        state: "basic"
     }
 } satisfies Story;
-toastWithoutHeader.parameters = { controls: { include: ['headerTitle', 'autohide', 'withIcon', 'delay', 'showHeader', 'message', 'colorVariant', 'iconName', 'iconColorvariant', 'iconHeight', 'iconWidth', 'iconFill'] } };
+toastWithRequest.parameters = { controls: { include: ["headerTitle", "message", "delay", "autohide", "withIcon", "showHeader", "iconName", "iconHeight", "iconWidth", "iconFill", "borderColor", "layout", "state"] } };
+
+// export const toastWithAutohide: Story = {
```

Comment:
remove commented code if not required in future

Reply:
removed commented code which is not required.
in rds-toasts.tsx file there are 2-3 commented lines which are earlier used for color variant changes so kept that for reference.

Reply:
@hemalatagaikwad @DeepashriDatar check this

Reply:
resolved